packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -59,6 +61,7 @@
     $: formattedStakedAmount = formatTokenAmountBestMatch(rawStakedAmount, getBaseToken())
     $: primaryKey = $selectedWallet?.primaryKey
     $: listBlockKeysFeature(selectedOutput)
+    $: hasMainAccountNegativeBIC = $selectedWallet?.balances?.blockIssuanceCredits[$selectedWallet?.mainAccountId] < 0
 
     function getAccountBalance(outputData: OutputData, isImplicitAccount: boolean): number | undefined {
```

Comment:
`getAccountBalance` -> `getImplicitAccountBalance` and remove the `if` from `account` because this is not going to be done, we can remove the code